sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md

```diff
@@ -8,112 +8,313 @@ The authentication events trigger for Azure Functions allows you to implement a
+2. Press **F5** or select *AuthEventsTrigger* from the top menu to run the function. 
+3. Copy the **Function url** from the terminal that popups up when running the function. This can be used when setting up a custom authentication extension.
 
-* Visual Studio 2019
-  * Start Visual Studio
-  * Select "Create a new project"
-  * In the template search area search and select "AzureAuthEventsTrigger"
-  * Give your project a meaningful Project Name, Location, Solution and Solution Name.
+#### [Visual Studio Code](#tab/visual-studio-code)
 
-* Visual Studio Code
-  * Start Visual Studio Code
-  * Run the command "Create Azure Authentication Events Trigger Project" via the command palette
-  * Follow the project creation prompts
-* Please note: that on a first time run it might take awhile to download the the required packages.
-* For development purpose turn of token validation for testing:
-* Add the **AuthenticationEvents__BypassTokenValidation** application key to the "Values" section in the local.settings.json file and set it's value to **true**. If you do not have a local.settings.json file in your local environment, create one in the root of your Function App.
+1. In the top menu, select **Run** > **Start Debugging** or press **F5** to run the function.
+2. In the terminal, copy the **Function url** that appears. This can be used when setting up a custom authentication extension.
+
+---
+
+### Run the function locally (recommended)
+
+It's a good idea to test the function locally before deploying it to Azure. We can use a dummy JSON body that imitates the request that Microsoft Entra ID sends to your REST API. Use your preferred API testing tool to call the function directly.
+
+1. In your IDE, open *local.settings.json* and replace the code with the following JSON. We can set `"AuthenticationEvents__BypassTokenValidation"` to `true` for local testing purposes.
 
 ```json
 {
   "IsEncrypted": false,
   "Values": {
-    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+    "AzureWebJobsStorage": "",
+    "AzureWebJobsSecretStorageType": "files",
     "FUNCTIONS_WORKER_RUNTIME": "dotnet",
-    "AuthenticationEvents__BypassTokenValidation": true
+    "AuthenticationEvents__BypassTokenValidation" : true
   }
 }
 ```
 
-* Once the project is loaded, you can run the sample code and you should see the Azure functions developer's application load your end point.
+2. Using your preferred API testing tool, create a new HTTP request and set the **HTTP method** to `POST`.
+3. Use the following JSON body that imitates the request Microsoft Entra ID sends to your REST API.
+
+```json
+{
+    "type": "microsoft.graph.authenticationEvent.tokenIssuanceStart",
+    "source": "/tenants/aaaabbbb-0000-cccc-1111-dddd2222eeee/applications/00001111-aaaa-2222-bbbb-3333cccc4444",
+    "data": {
+        "@odata.type": "microsoft.graph.onTokenIssuanceStartCalloutData",
+        "tenantId": "aaaabbbb-0000-cccc-1111-dddd2222eeee",
```

Comment:
These are going to get flagged as false positives for security issues.    Please use either the empty GUID (all zeros) or a completely nonsense value.

Reply:
Updated guids to what the readme payload example had before